lua/pears/presets/tag_matching.lua

```diff
@@ -33,6 +37,16 @@ return function(conf, opts)
       R.when(
         R.lang {"typescript", "tsx"},
         R.not_(R.start_of_context "[a-zA-Z0-9]")),
+      -- Don't expand for tags configured to be skipped
+      R.not_(
+        R.any_of(
+          unpack(vim.tbl_map(function (tag)
+              return R.end_of_context(tag.."[^>]*")
```

Comment:
I'm not sure if this will work if the tag has attributes... I need to test it. We may need to check from the start of the context instead of the end. `R.start_of_context(tag)`?

Reply:
For example `<input type="text">`

Reply:
It does work when the tag has attributes. In fact, if I change it to use `start_of_context`, it no longer works.